changelog/1.28.1.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "1.28.1"
+description: "Released on 02/23/2022"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.28.1.
+
+### Features ✨
+
+There are no new features in 1.28.1.
+
+### Bug fixes 🐛
+
+- infra: fixed issue regarding mTLS not working with Git providers.
+- infra: fixed issue regarding mTLS not working with Docker registries.
```

Comment:
These could be condensed to "not working with Git providers and Docker registries"